File: Inter-Process/pescaFortunata/java/src/Console.java
public class Console {
    private String nome;
    private boolean lettoreDisco;

    public Console(String nome, boolean lettoreDisco) {
        this.nome = nome;
        this.lettoreDisco = lettoreDisco;
    }

    public String toString() {
        return "Console: " + nome + (lettoreDisco ? " con lettore disco" : " senza lettore disco");
    }
}

File: Inter-Process/pescaFortunataV2/Console.py
class Console:
    def __init__(self, nome, lettore_disco):
        self.nome = nome
        self.lettore_disco = lettore_disco

    def __str__(self):
        return f"Console: {self.nome} {'con lettore disco' if self.lettore_disco else 'senza lettore disco'}"